tools/src/h5diff/h5diff_common.c

```diff
@@ -816,7 +816,7 @@ usage(void)
     PRINTVALSTREAM(rawoutstream, "  2) Datasets\n");
     PRINTVALSTREAM(rawoutstream,
                    "      Array rank and dimensions, datatypes, and data values are compared.\n");
-    PRINTVALSTREAM(rawoutstream, "  3) Datatypes\n");
+    PRINTVALSTREAM(rawoutstream, "  3) Attributes and Datatypes\n");
```

Comment:
Should this group attributes and datasets instead of attributes and datatypes?

Reply:
yes!!